tests/test_posting_models.py

```diff
@@ -358,6 +358,16 @@ def test_fix_bug_of_weight_using_diameter_information():
     assert package.real_diameter == 2
 
 
+@pytest.mark.parametrize('package_type,diameter', [
+    ('TYPE_ENVELOPE', 0),
+    ('TYPE_BOX', 0),
+    ('TYPE_CYLINDER', 16),
+])
+def test_package_diameter(package, package_type, diameter):
```

Comment:
I think you should add a paremeter with diameter larger than `MIN_DIAMETER` to test the second condition of `max()` call.